example/github/main.tf

```diff
@@ -20,4 +20,8 @@ resource "github_branch" "main" {
 resource "github_branch" "development" {
   repository = github_repository.repository.name
   branch     = "development"
-}
\ No newline at end of file
+}
+
+output "repository_name" {
+  value = github_repository.repository.name
+}
```

Comment:
Previously, the tests did not exercise `tf-summarize`'s ability to summarize output changes. Now they do!